lib/zebra/zpl/text.rb

```diff
@@ -66,8 +66,8 @@ def to_zpl
         if bold.nil?
           "^FW#{rotation}^A#{font_type},#{font_size}^CI28^FO#{x},#{y}^FB#{width},#{max_lines},#{line_spacing},#{justification},#{hanging_indent}#{appear_in_reverse}^FD#{data}^FS"
         else
-          "^FW#{rotation}^A#{font_type},#{font_size}^CI28^FO#{x+2},#{y}^FB#{width},#{max_lines},#{line_spacing},#{justification},#{hanging_indent}#{appear_in_reverse}^FD#{data}^FS" +
```

Comment:
In the original Gem we have x + 2 / y + 2 - but for SH usage it needs to be a bit tinier